app/reducers/transaction.js

```diff
@@ -230,11 +232,17 @@ transactionsSelectors.transactionsSelector = createSelector(
         const channelData = getChannelData(channelObj)
         return [channelData.closing_tx_hash, channelObj.closing_txid].includes(transaction.tx_hash)
       })
+      const pendingChannels = [...closingPendingChannelsRaw, ...pendingOpenChannelsRaw].find(
```

Comment:
the name of the constant is `pendingChannelS` but find returns a single value (or non at all), so probably should be named `pendingChannel`

Reply:
Updated

Reply:
Updated.